src/benchmarks/micro/libraries/System.Text.Json/Serializer/ReadJson.cs

```diff
@@ -57,6 +57,14 @@ public async Task Setup()
         [Benchmark]
         public T DeserializeFromUtf8Bytes() => JsonSerializer.Deserialize<T>(_utf8Serialized);
 
+        [BenchmarkCategory(Categories.Libraries, Categories.JSON)]
```

Comment:
nit: this attribute could be applied to the entire class and then there would be no need to apply it to each benchmark separately

Reply:
I figured I'd just stay consistent with what we already have rather than change it right now.